pkg/virt-controller/watch/topology/filter.go

```diff
@@ -89,3 +92,8 @@ func FilterNodesFromCache(objs []interface{}, predicates ...FilterPredicateFunc)
 	}
 	return match
 }
+
+// ToleranceForFrequency returns 250 parts per million of freq, rounded down to the nearest Hz
```

Comment:
nit:
instead of 250 you can use `TSCTolerancePPM` const. It will be easier to maintain.